### ISSUE TITLE
I want a rule that will warn about multiple parametrize fixtures for the same param

### ISSUE THREAD

**Issue by webknjaz** [m-burst/flake8-pytest-style#38]
# Rule request

## Description

I just got a PR with a change like this:
```python
@pytest.mark.parametrize('param1', ...)
@pytest.mark.parametrize('param1', ...)
@pytest.mark.parametrize('param1', ...)
def test_...()...
```

Clearly the person how wrote it doesn't understand how it works. But then I thought that since multiple parameterized decorations are allowed, maybe it makes sense to lint for silly mistakes like this.

## Rationale

It's for preventing typos and helping beginners to deal with rookie mistakes.

**Comment by m-burst** [comment]
Hi @webknjaz,

I can see that pytest reports this as an error during test collection.
Not sure whether we need to replicate this check in a linter rule.
I suppose if someone didn't bother with running the tests locally, they wouldn't have run the linters either, so the problem would have been found in the CI anyway.

**Comment by webknjaz** [comment]
Fair enough.
